contrib/hyperkube-image/Makefile

```diff
@@ -0,0 +1,20 @@
+TAG = v1.1.2
+PREFIX = quay.io/coreos
+
+.PHONY: all container push clean
+
+all: container
+
+hyperkube:
+	#TODO(aaron): build from source?
```

Comment:
Yes, aren't we building our own containers for tectonic releases? If not we will need to at some point to do hot fixes.